src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(Map<String, Integer> storage, FruitTransaction transaction) {
```

Comment:
storage imitates real DB. So you should make it a public static final map of the Storage class